aex_accounts/version.py

```diff
@@ -0,0 +1,9 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator 1.0.1.0
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+VERSION = "4.0"
```

Comment:
i don't understand how this version is related to Accounts. Should this be defined somewhere else?

Reply:
This is AEX API version which is being used in setting up user-agent name.

Reply:
we decided to change it to azurecli/<pypi-package-version>